popoolation2.rb

```diff
@@ -0,0 +1,101 @@
+class Popoolation2 < Formula
+  desc "Allows to compare allele frequencies for SNPs between two or more populations and to identify
+        significant differences. PoPoolation2 requires next generation sequencing data of pooled genomic
+        DNA (Pool-Seq). It may be used for measuring differentiation between populations, for genome wide
+        association studies and for experimental evolution."
+  homepage "https://sourceforge.net/p/popoolation2/wiki/Main/"
+  url "https://downloads.sourceforge.net/project/popoolation2/popoolation2_1201.zip"
+  version "1.2.01"
+  sha256 "7f45353a211ec88d36fd1beed1d128673f3ff911b731211eb5a39f8602bddd42"
+
+  bottle :unneeded
+
+  depends_on "cpanminus" => :build
+  depends_on :perl => ["5.8", :run]
+  depends_on :java => ["1.8", :run, :recommended]
```

Comment:
Is the `:recommended` option necessary here? If it refers to the version, it is alright; but if it refers to java, it is required for running `mpileup2sync.jar` because you are removing the perl version. You can work around this by removing the `mpileup2sync.java` if there is no java and keep the `mpileup2sync.pl`instead. Nevertheless, I think that it is better to just require java.

Reply:
Good point.